src/shared/killall.c

```diff
@@ -257,7 +257,7 @@ static int killall(int sig, Set *pids, bool send_sighup) {
 
                 r = pidref_kill(&pidref, sig);
                 if (r < 0) {
-                        if (errno != -ESRCH)
+                        if (errno != ESRCH)
                                 log_warning_errno(errno, "Could not kill " PID_FMT ", ignoring: %m", pidref.pid);
```

Comment:
No, `pidref_kill()` is our function, hence please use negative errno:
```c
if (r < 0) {
        if (r != -ESRCH)
                log_warning_errno(r, "...");
}
```

Reply:
Ouch, ouch.